Documentation/DocuBlocks/Rest/Views/post_api_view_iresearch.md

```diff
@@ -13,7 +13,7 @@ The type of the view. must be equal to *"arangosearch"*
 The view properties. If specified, then *properties* should be a JSON object
 containing the following attributes:
 
-@RESTSTRUCT{cleanupIntervalStep,post_api_view_props,integer,optional,uint64}
```

Comment:
are these changes meant to be in this PR ?

Reply:
Fixed. Was a side-effect of us having a 3.5.1 branch and this PR being based on 3.5